db/schema.rb

```diff
@@ -0,0 +1,10 @@
+ActiveRecord::Schema.define(version: 20160719100630) do
+  enable_extension "plpgsql"
+
+  create_table "wishlists", force: :cascade do |t|
+    t.string   "name"
+    t.text     "description"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+end
```

Comment:
You probably will have some kind validation? Should be added to model and migration.